scripts/linux/runE2ETest.sh

```diff
@@ -979,7 +987,9 @@ LOADGEN2_TRANSPORT_TYPE="${LOADGEN2_TRANSPORT_TYPE:-amqp}"
 LOADGEN3_TRANSPORT_TYPE="${LOADGEN3_TRANSPORT_TYPE:-mqtt}"
 LOADGEN4_TRANSPORT_TYPE="${LOADGEN4_TRANSPORT_TYPE:-mqtt}"
 if [[ "${TEST_NAME,,}" == "longhaul" ]]; then
+    DESIRED_MODULES_TO_RESTART_CSV="${DESIRED_MODULES_TO_RESTART_CSV:-moduleRestarter}"
```

Comment:
why default value is "moduleRestarter"?

Reply:
I did not want to pick any modules by default, as that could lead to unexpected behavior. I thought the cleanest solution was to restart itself so that we are still testing the the edgeAgent device method without having to restart any module involved in message/method/twin operations.